src/Workspaces/Core/Portable/Classification/IRemoteSemanticClassificationCacheService.cs

```diff
@@ -1,35 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Remote;
-using Microsoft.CodeAnalysis.Serialization;
-using Microsoft.CodeAnalysis.Storage;
-using Microsoft.CodeAnalysis.Text;
-
-namespace Microsoft.CodeAnalysis.Classification
-{
-    /// <summary>
-    /// Remote stubs used for a host to request cached semantic classifications from an OOP server.
-    /// </summary>
-    internal interface IRemoteSemanticClassificationCacheService
-    {
-        ValueTask CacheSemanticClassificationsAsync(
```

Comment:
this method is gone completely.  the client does not choose when things are cached.